ign-math7.yaml

```diff
@@ -1,14 +1,18 @@
 ---
 repositories:
-  ign-cmake:
+  ign-cmake2:
     type: git
     url: https://github.com/ignitionrobotics/ign-cmake
     version: ign-cmake2
```

Comment:
We have to remember to remove this one when we bump `ign-math7` to use `ign-cmake3`. We've never done it this way before, so the script won't be able to handle it. It will have to be done manually

Reply:
I added it to gz-cmake3 roadmap

Reply:
Thanks. I hope we catch this before we close https://github.com/ignition-tooling/release-tools/issues/685